napari/layers/_tests/test_layer_attributes.py

```diff
@@ -18,3 +19,24 @@ def test_layer_bounding_box_order(image_shape, dims_displayed, expected):
     assert np.allclose(
         layer._display_bounding_box(dims_displayed=dims_displayed), expected
     )
+
+
+@pytest.mark.parametrize('Layer, data, ndim', layer_test_data)
```

Comment:
I guess we could also parameterize by the attribute we update (e.g. `scale`, `data`), but maybe that's overkill and could be hard to follow.